File: TigerBook/python/one_undergrad.py
from req_lib import getOneUndergrad

if __name__ == "__main__":
    # Please put the netid you need for the
    # parameter
    req = getOneUndergrad(netid="vramesh")
    if req.ok:
        print(req.json())
    else:
        print(req.text)

File: TigerBook/java/OneUndergrad.java
public class OneUndergrad {
    public static void main(String[] args) {
        ReqLib reqLib = new ReqLib();

        try {
            /*
                Please put the netid you need for the
                parameter
            */
            String response = reqLib.getOneUndergrad("vramesh");
            System.out.println(response);
        } catch (Exception e) {
            throw new RuntimeException(e);
        }
    }
}